monitoring/repo_updater.go

```diff
@@ -154,21 +156,22 @@ func RepoUpdater() *Container {
 							PossibleSolutions: "Check repo-updater logs. This is expected to fire if there are no user added code hosts",
 						},
 						Observable{
-							Name:              "sched_update_queue_length",
-							Description:       "repositories queued for update",
-							Query:             `sum(src_repoupdater_sched_update_queue_length)`,
-							DataMayNotExist:   true,
-							Critical:          Alert().GreaterOrEqual(1000).For(5 * time.Minute),
+							Name:            "sched_update_queue_length_derivative",
+							Description:     "derivative of update queue length over 5 minutes",
```

Comment:
```suggestion
							Description:     "rate of growth of update queue length over 5 minutes",
```

(perhaps a bit more plain-language?)